### PR TITLE
fix(RadioGroup): Update examples to enable input for in the tab order inside Radio Group

### PR DESCRIPTION
**Pull request by assuncaocharles**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Update examples to enable input for in the tab order inside Radio Group

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/13110)

### CODE REVIEW

**Comment by silviuaavram** [comment]
This example is just so weird. We removed the input from the tab order just to put some custom keyboard handling in this PR.

Why was the input removed from the tab order from the first place?

**Comment by assuncaocharles** [comment]
> This example is just so weird. We removed the input from the tab order just to put some custom keyboard handling in this PR.
> 
> Why was the input removed from the tab order from the first place?
> 
> Edit: maybe because we are focusing the whole wrapper on tab instead of the radio button. Which is not a radio button. This looks just so different from what an `<input type="radio">` is.

I was also wondering if it was really a problem with the behavior or just a matter of the example that explicitly remove the input from tab order

File: packages/fluentui/accessibility/src/behaviors/Radio/radioGroupItemBehavior.ts [hunk 1]
```diff
@@ -29,6 +31,12 @@ const radioGroupItemBehavior: Accessibility<RadioGroupItemBehaviorProps> = props
       },
     },
   },
+
+  focusZone: {
+    props: {
+      shouldEnterInnerZone: event => keyboardKey.getCode(event) === keyboardKey.Tab,
```
Comment (jurokapsiar) [inline packages/fluentui/accessibility/src/behaviors/Radio/radioGroupItemBehavior.ts:37]: why is this needed? as far as I remember, radios do not use focus zone

Comment (assuncaocharles) [reply packages/fluentui/accessibility/src/behaviors/Radio/radioGroupItemBehavior.ts]: Reverted
